web/libs/datamanager/src/components/Common/Form/Validation/Validation.scss

```
@@ -0,0 +1,27 @@
+.form-validation {
+  margin-top: 32px;
+
+  &__group {
+    padding: 7px 14px;
+    border-radius: 5px;
+    background-color: rgba(#c00, 0.04);
+
+    &+& {
+      margin-top: 5px;
+    }
+  }
+
+  &__field {
+    font-size: 14px;
+    font-weight: bold;
+  }
+
+  &__messages {
+    margin-top: 3px;
+  }
+
+  &__message {
+    font-size: 12px;
+    color: rgba(#000, 0.7);
```

Flagging as opportunity for css var replacement. Should also be reviewed further for usage.
```suggestion
    color: rgba(var(--black), 0.7);
```